packages/confirm-dialog/src/vaadin-confirm-dialog.js

```diff
@@ -79,6 +79,12 @@ class ConfirmDialog extends ConfirmDialogMixin(ElementMixin(ThemePropertyMixin(P
       [hidden] {
         display: none !important;
       }
+
+      :host([opened]),
+      :host([opening]),
+      :host([closing]) {
+        display: contents !important;
+      }
```

Comment:
This is kind of a new concept, but we need it to hide the dialog from the accessibility tree when closed.

I also wonder if something like this should have been in the tooltip PR, as the tooltip now also shows up in the accessibility tree.